Marlin/src/gcode/calibrate/G33.cpp

```diff
@@ -71,7 +71,9 @@ float lcd_probe_pt(const xy_pos_t &xy);
 
 void ac_home() {
   endstops.enable(true);
+  TERN_(SENSORLESS_PROBING, probe.set_homing_current(true));
   home_delta();
+  TERN_(SENSORLESS_PROBING, probe.set_homing_current(false));
```

Comment:
is not 
TERN_(HAS_HOMING_CURRENT , probe.set_homing_current(false));
?

Reply:
The same as the previous .

Reply:
Idem.